exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 var studentCount = 16;
 var mentorCount = 9;
-var moreStudentsThanMentors; // finish this statement
+var moreStudentsThanMentors = true;
 
 var roomMaxCapacity = 25;
-var enoughSpaceInRoom; // finish this statement
+var enoughSpaceInRoom = true;
 
 var personA = "Daniel";
 var personB = "Irina";
-var sameName; // finish this statement
+var sameName = false;
 
```

Comment:
Hi Kuda in this exercise you are suppose to use comparison operators in these statements so that it can return a boolean value. Comparison operators(>, <, ===, !==).

Reply:
Sorry I mean Erick

Reply:
ohh, i see what you mean, thank you